src/graph.rs

```diff
@@ -988,4 +1149,145 @@ mod tests {
 
         assert_eq!(graph.index.as_ref().unwrap().len(), 2);
     }
+
+    #[test]
+    fn randomish_graph() {
```

Comment:
these cases might deserve their own test module. Otherwise I'd recommend making it clear that they all test closeness/betweenness as this isn't clear from the output:
``` 
test graph::tests::randomish_graph ... ok
test graph::tests::star_graph_a ... ok
test graph::tests::star_graph_b ... ok
```

Reply:
done